package.json

```diff
@@ -63,6 +63,7 @@
     "common-path-prefix": "^3.0.0",
     "cp-file": "^10.0.0",
     "es-module-lexer": "^1.0.0",
+    "esbuild": "^0.19.2",
```

Comment:
Do we want to restrict it to the patch version as esbuild does breaking changes in minor versions? like `~0.19.2`

Reply:
That's a good idea! I'll lock it down to an exact version, no need of ambiguity here.

Reply:
Done in [2dd12bb](https://github.com/netlify/zip-it-and-ship-it/pull/1532/commits/2dd12bb17043b3521a81123c7c5b3c4f0aa6b5b9).